plugin/src/software/aws/toolkits/eclipse/amazonq/lsp/editor/ActiveEditorChangeListener.java

```diff
@@ -62,20 +67,33 @@ public void stop() {
 
     @Override
     public void partActivated(final IWorkbenchPartReference partRef) {
-        if (partRef.getPart(false) instanceof ITextEditor) {
-            ITextEditor editor = (ITextEditor) partRef.getPart(false);
-            handleEditorChange(editor);
+        Object part = partRef.getPart(false);
```

Comment:
instead of a defining a type, you can simplify and declare this as a `var` which allows java to auto-resolve a type based on the evaluation of the expression on the right side.